internal/command/arguments/show.go

```diff
@@ -153,11 +170,16 @@ func ParseShow(args []string) (*Show, tfdiags.Diagnostics) {
 		show.TargetType = ShowConfig
 		show.TargetArg = ""
 	}
+	if moduleTarget != "" {
+		targetTypes++
+		show.TargetType = ShowModule
+		show.TargetArg = moduleTarget
+	}
 	if targetTypes != 1 {
 		diags = diags.Append(tfdiags.Sourceless(
 			tfdiags.Error,
 			"Conflicting object types to show",
-			"The -state, -plan=FILENAME, and -config options are mutually-exclusive, to specify which kind of object to show.",
+			"The -state, -plan=FILENAME, -config, and -module=DIR options are mutually-exclusive, to specify which kind of object to show.",
```

Comment:
```suggestion
			"The -state, -plan=FILENAME, -config and -module=DIR options are mutually-exclusive, to specify which kind of object to show.",
```
Typo?

Reply:
Wow, that's a rabbit hole... 🤣 Thanks for sharing.
Resolving this. My english skills are not even close to what it's needed to get into such an argument... 😅